start.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+nix-shell default.nix
```

Comment:
I know you can drop it, it helped me to figure out what's going on, as a newbie. I prefer explicit over implicit.